src/plugins/cache/cache.c

```diff
@@ -267,16 +289,29 @@ int elektraCacheGet (Plugin * handle ELEKTRA_UNUSED, KeySet * returned, Key * pa
 		return ELEKTRA_PLUGIN_STATUS_SUCCESS;
 	}
 	// get all keys
-
 	CacheHandle * ch = elektraPluginGetData (handle);
 	if (ch->cacheStorage->global == 0)
 	{
 		ch->cacheStorage->global = elektraPluginGetGlobalKeySet (handle);
 	}
 
+	if (!elektraStrCmp (keyString (keyGetMeta (parentKey, "cacheClear")), "YES"))
```

Comment:
cacheClear is not according our conventions, it should be cache/clear.

Furthermore it should be documented in doc/METADATA.ini